compiler-rt/src/alu/abs/abs_i40.cairo

```diff
@@ -0,0 +1,156 @@
+use crate::alu::abs::abs;
+use crate::integer::u40::u40;
+
+pub fn __llvm_abs_i40_i40(arg: u128, _is_int_min_poison: u128) -> u128 {
```

Comment:
This is not the correct naming scheme. It should be `__llvm_abs_n_c_n`.

Reply:
I think it is `__llvm_abs_n_n_n`